DataFormats/TrackReco/interface/TrackBase.h

```diff
@@ -816,6 +855,18 @@ inline double TrackBase::dzError() const
     return error(i_dsz) * p() / pt();
 }
 
+// error on t0
+inline double TrackBase::t0Error() const
+{
+    return std::sqrt(covt0t0_);
+}
+
+// error on beta
+inline double TrackBase::betaError() const
+{
+    return std::sqrt(covbetabeta_);
```

Comment:
there should also be a way to get these without sqrt.
Perhaps covarianceT0T0 and covarianceBetaBeta.
BTW, are the two strictly related by a track-dependent constant or is there a T0Beta term?